pokemongo_bot/cell_workers/sniper.py

```diff
@@ -331,6 +332,10 @@ def snipe(self, pokemon):
                 if (seconds_since_last_check < self.MIN_SECONDS_ALLOWED_FOR_CELL_CHECK):
                     time.sleep(self.MIN_SECONDS_ALLOWED_FOR_CELL_CHECK - seconds_since_last_check)
 
+                # Sleep a bit longer for the Pokemon to appear
+                self._log('Waiting for the Pokemon to appear...')
+                time.sleep(randint(5,10))
```

Comment:
I tested with 2 bots:
# this one sleeps 3 secs (by default) and missed the Pokemon
[2016-09-26 20:19:52] [MoveToFort] [INFO] Moving towards pokestop USPS Sign - 0.05km
[2016-09-26 20:19:57] [    Sniper] [INFO] Teleporting to meet Growlithe (41.7646115485; -72.9637643552)...
[2016-09-26 20:19:57] [    Sniper] [INFO] Sleeping 3 secs during teleporting...
[2016-09-26 20:20:00] [    Sniper] [INFO] Damn! Its not here. Reasons: too far, caught, expired or fake data. Skipping...
[2016-09-26 20:20:00] [    Sniper] [INFO] Teleporting back to the old position (36.1876341128; -122.986435657)...

#This one sleeps a little bit more and caught the same exact Pokemon
[2016-09-26 20:20:06] [    Sniper] [INFO] Teleporting to meet Growlithe (441.7646115485; -72.9637643552)...
[2016-09-26 20:20:06] [    Sniper] [INFO] Sleeping 3 secs during teleporting...
[2016-09-26 20:20:09] [    Sniper] [INFO] Waiting 8 secs for the Pokemon to appear...
[2016-09-26 20:20:17] [    Sniper] [INFO] Yay! There really is a wild Growlithe nearby!
[2016-09-26 20:20:17] [    Sniper] [INFO] Teleporting back to the old position (36.1250140604; -121.805482928)...
[2016-09-26 20:20:17] [    Sniper] [INFO] Sleeping 3 secs during teleporting...
[2016-09-26 20:20:20] [PokemonCatchWorker] [INFO] *A wild Growlithe appeared!* (CP: 1086) (NCP: 0.8) (Potential 1.0) (A/D/S 15/15/15)
[2016-09-26 20:20:23] [PokemonCatchWorker] [INFO] This is a VIP pokemon. Catch!!!
[2016-09-26 20:20:26] [PokemonCatchWorker] [INFO] Threw a Razz Berry! Catch rate with Pokeball is now: 25.46
[2016-09-26 20:20:31] [PokemonCatchWorker] [INFO] Great throw! Used Ultraball, with chance 46.60 (15 left)
[2016-09-26 20:20:32] [PokemonCatchWorker] [INFO] Captured Growlithe! [CP 1086] [NCP 0.8] [Potential 1.0] [15/15/15] (7/800) [+150 exp] [+100 stardust]
[2016-09-26 20:20:32] [PokemonCatchWorker] [INFO] You now have 603 Growlithe candy!